### ISSUE TITLE
Color contrast is NaN:1

### ISSUE THREAD

**Issue by AntonOellerer** [pa11y/pa11y-ci#124]
Hey,
First of all, thank you for this great library!
I am currently trying to adapt our project to the WCAG2AAA standard, and locally managed to fix all of the problems.
Now I pushed it to our repository, and for a reason I can not explain the testing in the GitLab CI pipeline fails with messages similar to
```
 • This element has insufficient contrast at this conformance level. Expected a
   contrast ratio of at least 4.5:1, but text in this element has a contrast
   ratio of NaN:1.
   (#main > section:nth-child(2) > div > div > h1)
   <h1 class="title">Alpha version</h1>
```
What especially confuses me about this is the `NaN:1` ratio.
Do you know what the reason for this could be?
Best regards
Anton Oellerer

**Comment by AntonOellerer** [comment]
Ok it seems as if this was related to a banner we only showed in productive builds, I will move this issue to pa11y, sorry for the confusion.
